services/catarse/catarse.js/legacy/src/c/project-data-table.tsx

```diff
@@ -90,7 +90,7 @@ const projectDataTable = {
                 _.map(body, rowData => m('.w-row.table-row',
                         _.map(rowData, (row) => {
                             // Check if a custom comparator is used => Read component description
-                            row = (_.isArray(row) && row.length > 1) ? row[1] : row;
+                            row = (_.isArray(row) && row.length > 1) ? [].concat(row[1].map(obj => obj)) : row;
```

Comment:
**Sugestão:** Não consegui entender aqui @gabrielras. Tem como lançar uma explicação simples ali no código com um comentário.

**Sugestão:**
Talvez para garantir que é um array nesse caso true do operador ternário pode fazer assim: `Array.of(row[1]).filter(d => d)`.

E se puder modificar o nome ou colocar em uma variável temporária com o significado dessa mudança já fica melhor para fazermos manutenção depois.

Reply:
Show, acho que um comentário vai ajudar a explicar que quando vc customiza uma coluna é preciso concatenar para que funcione corretamente no componente.
Sobre o uso do Array.of, ele acaba não funcionando corretamente no componente pq a estrutura é diferente.
Array.of(row[1]).filter(d => d) retorna:
```
[Array(4)]
0:
attrs: undefined
children: (4) [{…}, {…}, {…}, {…}]
```
Usando [].concat(row[1].map(obj => obj)) retorna:
```
(4) [ "R$ ", "1.860,00", " (18.58%)"]
1: {tag: "#", key: undefined, attrs: undefined, children: "R$ ", text: undefined, …}
2: {tag: "#", key: undefined, attrs: undefined, children: "1.860,00", text: undefined, …}
3: {tag: "#", key: undefined, attrs: undefined, children: " (18.58%)", text: undefined, …}
```